Firestore/Source/Local/FSTWriteGroup.h

```diff
@@ -46,13 +47,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (instancetype)groupWithAction:(NSString *)action;
 
++ (instancetype)groupWithAction:(NSString *)action
+                 andTransaction:(firebase::firestore::local::LevelDBTransaction *)transaction;
```

Comment:
Don't use joining words on arguments beyond the first. Just `groupWithAction:transaction:`.

Reply:
Done.